Makefile

```diff
@@ -246,3 +246,16 @@ help:
 	@echo ""
 	@echo "Help:"
 	@echo "  help           Show this help message"
+
+################################################################################
```

Comment:
For both cypress and jest  `docker-compose` has been used instead of `docker compose`.
Commit#ed5fbd3b2d74fa658bec810e36ff689d67a41198 corrects this